client/src/GameState.ts

```diff
@@ -572,6 +578,17 @@ export class GameState {
         return true;
     }
 
+    private enteringNeighborPrivateArea(
```

Comment:
can we name it to `isInNeighborSection` or something along those lines to match it up with `isInOppositeSection`